src/models/source_code_unit.rs

```diff
@@ -339,9 +339,12 @@ impl SourceCodeUnit {
     // Get the tree_sitter's input edit representation
     let (new_source_code, ts_edit) = get_tree_sitter_edit(self.code.clone(), &edit_to_apply);
     // Apply edit to the tree
+    let number_of_errors = self._number_of_errors();
```

Comment:
It could maybe make sense to have this thing configurable via the command line? 
`--allow-dirty-tree`

Reply:
Wait. Not sure I understand this. Are you saying this is not configurable through the command line? I thought the whole point of PiranhaArguments was a similar interface between CLI and programatic invocation... what am I missing? (Or you mean having a different argument that allows the number of errors to increase?)